### PR TITLE
Forecast: Swapped around temperature and icon + fixed styling

### PR DESCRIPTION
**Pull request by NeoSilky**
**What did I fix?**:

I fixed issue #1546 where the orientation of the text and icon could cause problems.

cc @jagtalon

### CODE REVIEW

File: share/spice/forecast/forecast_detail_mobile.handlebars [hunk 1]
```diff
@@ -5,10 +5,10 @@
 		<div class="fe_current-wrap">
 		{{#with current}}
 			<div class="fe_current__top">
-			  <span class="fe_icon-wrap  fe_icon-wrap--current">
+			  <h1 class="fe_temp  tx-clr--dk">{{{temp}}}</h1>
+              <span class="fe_icon-wrap  fe_icon-wrap--current">
```
Comment (moollaza) [inline share/spice/forecast/forecast_detail_mobile.handlebars]: Looks like the indentation is a little wonky, please fix :)

**Comment by NeoSilky** [comment]
@moollaza Fixed :)

**Comment by jagtalon** [comment]
@NeoSilky cool thanks! Will just test this on diff browsers and it should be good to go.

**Comment by jagtalon** [comment]
Looking good! :+1:

**Pull request merged**